src/main/scala/it/pagopa/interop/apigateway/api/impl/package.scala

```diff
@@ -194,7 +163,11 @@ package object impl extends SprayJsonSupport with DefaultJsonProtocol {
 
   implicit class EnrichedEService(private val eService: CatalogManagementApiEService) extends AnyVal {
     def latestAvailableDescriptor: Future[CatalogManagementApiDescriptor] =
-      eService.descriptors.sortBy(_.version.toInt).lastOption.toFuture(MissingAvailableDescriptor(eService.id.toString))
+      eService.descriptors
+        .filter(_.state != CatalogManagementApiDescriptorState.DRAFT)
```

Comment:
Bug

This is used in `enhanceEService`

Use case: 
- an organization performs a legit retrieve of another organization eservice(s)
- one or more returned eservices contain a descriptor draft
- the request fails because [here](https://github.com/pagopa/interop-be-api-gateway/pull/133/files#diff-45c4cb111d08dd8f28095838b0a597b24d6249d2241581517ee286af10975124R153) the DRAFT state cannot be returned